corehq/apps/toggle_ui/views.py

```diff
@@ -357,16 +375,7 @@ def set_toggle(request, toggle_slug):
     item = request.POST['item']
     enabled = request.POST['enabled'] == 'true'
     namespace = request.POST['namespace']
-    if static_toggle.set(item=item, enabled=enabled, namespace=namespace):
-        action = ToggleAudit.ACTION_ADD if enabled else ToggleAudit.ACTION_REMOVE
-        ToggleAudit.objects.log_toggle_action(
-            toggle_slug, request.user.username, [namespaced_item(item, namespace)], action
-        )
-
-    if enabled:
-        _notify_on_change(static_toggle, [item], request.user.username)
-
-    _clear_cache_for_toggle(namespace, item)
```

Comment:
I notice that the indent level for these last two blocks changed when you moved them, but I think the new behavior is more correct, so I'm guessing that was intentional.

Reply:
Yes, these are only needed if the toggle was actually updated.